qa/workunits/rados/test_envlibrados_for_rocksdb.sh

```diff
@@ -31,15 +31,18 @@ case $(distro_id) in
         esac
 		;;
 	centos|fedora|rhel)
-	        # el8 needs PowerTools for snappy-devel
-	        test -x /usr/bin/dnf && sudo dnf config-manager --set-enabled PowerTools || true
-		install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64
-        if [ $(distro_id) = "fedora" ]; then
-            install cmake
-        else
-            install_cmake3_on_centos7
-        fi
-		;;
+        case $(distro_id) in
+            centos)
+                # centos needs PowerTools repo for snappy-devel
+	            test -x /usr/bin/dnf && sudo dnf config-manager --set-enabled PowerTools || true
+                ;;
+            rhel)
+                # RHEL needs CRB repo for snappy-devel
+                sudo subscription-manager repos --enable "codeready-builder-for-rhel-8-x86_64-rpms"
```

Comment:
i have 
```
2019-12-27T01:43:29.445 INFO:tasks.workunit.client.0.smithi070.stderr:+ sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
2019-12-27T01:43:29.445 INFO:tasks.workunit.client.0.smithi070.stdout:Install required tools
2019-12-27T01:43:35.191 INFO:tasks.workunit.client.0.smithi070.stdout:Internal Server Error): Required lock is already taken by other running tasks.
2019-12-27T01:43:35.191 INFO:tasks.workunit.client.0.smithi070.stdout:Please inspect their state, fix their errors and resume them.
2019-12-27T01:43:35.191 INFO:tasks.workunit.client.0.smithi070.stdout:
2019-12-27T01:43:35.192 INFO:tasks.workunit.client.0.smithi070.stdout:Required lock: update
2019-12-27T01:43:35.192 INFO:tasks.workunit.client.0.smithi070.stdout:Conflicts with tasks:
2019-12-27T01:43:35.192 INFO:tasks.workunit.client.0.smithi070.stdout:  https://satellite.front.sepia.ceph.com/foreman_tasks/tasks/72d1b308 0b67 435e a947 c3da73c02dcf
2019-12-27T01:43:35.192 INFO:tasks.workunit.client.0.smithi070.stdout:
2019-12-27T01:43:35.192 INFO:tasks.workunit.client.0.smithi070.stdout:Error: 'codeready-builder-for-rhel-8-x86_64-rpms' does not match a valid repository ID. Use "subscription-manager repos --list" to see valid repositories.
```
in http://qa-proxy.ceph.com/teuthology/kchai-2019-12-27_01:28:11-rados-wip-kefu-testing-2019-12-27-0026-distro-basic-smithi/4632070/teuthology.log . 

@djgalloway any idea how to fix this failure? shall we exclude RHEL8 from this test?

Reply:
I can add this repo to the satellite.  Give me a few hours.

Reply:
thanks David!